cmd/podman/runlabel.go

```diff
@@ -146,10 +147,26 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 		return err
 	}
 	if !c.Quiet {
-		fmt.Printf("Command: %s\n", strings.Join(cmd, " "))
+		fmt.Printf("command: %s\n", strings.Join(cmd, " "))
 		if c.Display {
 			return nil
 		}
 	}
+
+	// If container already exists && --replace given -- Nuke it
+	if c.Replace {
+		for i, entry := range cmd {
+			if entry == "--name" {
+				ctr, err := runtime.LookupContainer(cmd[i+1])
+				if err == nil {
```

Comment:
`if err != nil && errors.Cause(err) != libpod.ErrNoSuchCtr` should at the very least log the error